### DISCUSSION TITLE
How to modify the `Tags` page?

### DISCUSSION THREAD

**Discussion by mrdanielmartins** [adityatelange/hugo-PaperMod#1217] [Q&A]
Hello Papermod Gang,

How does one modify the tags page? I tried to manually create a tags page but it overwrites the current one.

Thank you,

Daniel

**Comment by RoneoOrg** [comment]
> I tried to manually create a tags page but it overwrites the current one.

This is the expected behavior, what are you precisely trying to achieve, and where do you face a problem?

(publishing your code somewhere could help you get a proper answer)
